examples/vue/src/views/Home.vue

```diff
@@ -93,6 +94,10 @@ const updateSort = e => {
   initialFetch()
 }
 
+const productClick = product => {
+  KlevuEvents.searchProductClick(product, quickSearchStore.searchTerm)
```

Comment:
Instead of `KlevuEvents` we could use the new events system here and in Search.vue